nilearn/_utils/niimage.py

```diff
@@ -0,0 +1,127 @@
+import copy
+import gc
+import collections
+
+import nibabel
+
+
+def _get_shape(img):
```

Comment:
You mean that it is sometimes used from `niimg_conversion` module? I've seen that in the tests, I will fix it.